src/SpikeCore/SpikeCore/MessageBus/IrcSendChannelMessage.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+
+namespace SpikeCore.MessageBus
```

Comment:
Can you please bubble this back through the modules?

Reply:
Done as part of fixing the build errors.